src/wolfgame.py

```diff
@@ -767,7 +767,15 @@ def replace(cli, nick, chan, rest):
                 cli.notice(nick, msg)
             return
     else:
-        target, _ = complete_match(rest[0], var.list_players() + list(var.VENGEFUL_GHOSTS.keys()))
+        pl = var.list_players() + list(var.VENGEFUL_GHOSTS.keys())
+        pll = [var.irc_lower(i) for i in pl]
+
+        lc_target, _ = complete_match(var.irc_lower(rest[0]), pll)
+
+        if lc_target != None:
```

Comment:
use `is not None` here and do not replace target for the else condition -- otherwise the error message is very unhelpful (aka remove the else condition)